src/Microsoft.Azure.EventHubs/Primitives/ClientEntity.cs

```diff
@@ -1,11 +1,15 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.Collections.Concurrent;
```

Comment:
Move under namespace section

Reply:
Ohh resharper move it automatically and I didn't aware of that, super sorry will configure to avoid that in a future I did with other configurations :) .